src/js/components/SidebarComponent.jsx

```diff
@@ -57,6 +59,10 @@ var SidebarComponent = React.createClass({
         </div>
         <SidebarHealthFilterComponent />
         <SidebarLabelsFilterComponent />
+        <div className="flex-row">
+          <h3 className="small-caps">Resources</h3>
```

Comment:
The heading should be part of the filter component as it is in sidebar health and labels filter component.